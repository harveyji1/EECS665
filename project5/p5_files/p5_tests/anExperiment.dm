fn: () int{
    hello: int = 1;
    return hello;
}

f: (c: int) int{
    return 1;
}

main: () void{
    a: int;
    a = f;
    a = f(1);
    a = fn;
    f = fn();
    fn = f();
}