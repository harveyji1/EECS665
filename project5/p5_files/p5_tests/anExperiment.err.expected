FATAL [12,9]-[12,10]: Invalid assignment operand
FATAL [14,9]-[14,11]: Invalid assignment operand
FATAL [15,5]-[15,6]: Invalid assignment operand
FATAL [16,10]-[16,13]: Function call with wrong number of args
FATAL [16,5]-[16,7]: Invalid assignment operand
Type Analysis Failed
